<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>08서울남산체 B</family>
      <pointsize>35</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS를 이용한 FOG 초기화 자동화 프로그램</string>
    </property>
   </widget>
   <widget class="QLabel" name="Latitude">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="Longitude">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heading</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="FOGOutput">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>441</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FogOutput_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FOG OutPut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Manual_transmit_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manual Input-Data Transmit Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Automatic_transmit_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Automatic GPS-Data Transmit Button</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPS_Heading">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heading</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPS_Latitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPS_Longitude">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>FOG Configuration Data From GPS</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>411</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Manual Input</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="GPS_orig_Output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS Original Data</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Lat_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="Long_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="Heading_text">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="GPS_LatText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="GPS_LongText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="GPS_HeadingText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FOG Output Clear Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FOGResetbutton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FOG Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GPSRecvbutton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS Recv </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
